assembl/graphql/idea.py

```diff
@@ -197,6 +200,10 @@ def resolve_title_entries(self, args, context, info):
     def resolve_body(self, args, context, info):
         return resolve_langstring(self.body, args.get('lang'))
 
+    def resolve_body_attachments(self, args, context, info):
+        # TODO:
```

Comment:
is something need to be added here ?

Reply:
when we will add attachments on this field (I added the resolver and the TODO to make it easier)